packages/react-charting/src/utilities/TooltipText.tsx

```diff
@@ -0,0 +1,271 @@
+import * as React from 'react';
+import { ITooltipHost, ITooltipProps, Tooltip, TooltipDelay } from '@fluentui/react/lib/Tooltip';
+import { Async, KeyCodes, getId, portalContainsElement } from '../Utilities';
+
+interface ITooltipTextProps {
+  /**
+   * Number of milliseconds to delay closing the tooltip, so that the user has time to hover over
+   * the tooltip and interact with it. Hovering over the tooltip will count as hovering over the
+   * host, so that the tooltip will stay open if the user is actively interacting with it.
+   */
+  closeDelay?: number;
+
+  /**
+   * Content to display in the Tooltip.
+   */
+  content?: string;
+
+  /**
+   * Length of delay before showing the tooltip on hover.
+   * @defaultvalue TooltipDelay.medium
+   */
+  delay?: TooltipDelay;
+
+  /**
+   * Additional properties to pass through for Tooltip.
+   */
+  tooltipProps?: ITooltipProps;
+
+  /**
+   * Additional properties to pass through for SVG <text>.
+   */
+  textProps?: React.SVGAttributes<SVGTextElement>;
+
+  /**
+   * Max width of text
+   */
+  maxWidth?: number;
+
+  /**
+   * Max height of text
+   */
+  maxHeight?: number;
+
+  /** */
+  wrapContent?: (content: string, id: string, maxWidth: number, maxHeight?: number) => boolean;
+}
+
+interface ITooltipTextState {
+  isTooltipVisible: boolean;
+}
+
+export class TooltipText extends React.Component<ITooltipTextProps, ITooltipTextState> implements ITooltipHost {
```

Comment:
How is this tooltip different than the FocusableTooltip that you created earlier.

Reply:
This works with SVG text element but the FocusableTooltip is only for HTML texts